### PR TITLE
Add XREnvironmentBlendMode

### PR DESCRIPTION
**Pull request by a2sheppy**
Add XREnvironmentBlendMode.json to cover this
enum type.

Source: 
* https://immersive-web.github.io/webxr-ar-module/#xrenvironmentblendmode-enum

### CODE REVIEW

File: api/XREnvironmentBlendMode.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
+{
+  "api": {
+    "XREnvironmentBlendMode": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/XREnvironmentBlendMode",
+        "support": {
+          "chrome": {
+            "version_added": false
```
Comment (jpmedley) [inline api/XREnvironmentBlendMode.json]: All Chrome and Android Chrome values should be "79".

**Comment by Elchi3** [comment]
@a2sheppy are you planning to follow up on this PR?

**Comment by a2sheppy** [comment]
@Elchi3 - Yes, tomorrow. Sorry for the delay.

**Comment by a2sheppy** [comment]
Or today. ;)

**Pull request merged**